code/src/components/NotFound.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const NotFound = () => {
```

Comment:
It doesn't look like your custom 404 page is showing on your deployed version, was that intentional?

Reply:
I think we had some issues with our 404 page too after deployment. I can't remember what we did to fix it though? It's set up the same way as ours, so that's not it... Need to check with our group.